ui/src/shared/components/TableGraph.js

```diff
@@ -205,31 +194,27 @@ class TableGraph extends Component {
     })
   }
 
-  calculateColumnWidth = columnSizerWidth => column => {
+  calculateColumnWidth = __ => column => {
```

Comment:
what's the `__` here?

Reply:
the `__` is there because it doesn't actually use that columnSizerWidth value to calculate the size. However, it is still necessary that it gets passed in so that the this function gets called when its supposed to.